app/views/petitions/_petition.json.jbuilder

```diff
@@ -0,0 +1,62 @@
+json.type "petition"
+json.id petition.id
+
+json.links do
+  json.self petition_url(petition, format: params[:format])
+end if defined?(is_collection)
+
+json.attributes do
+  json.action petition.action
+  json.background petition.background
+  json.additional_details petition.additional_details
+  json.state petition.state
+  json.signature_count petition.signature_count
+
+  json.created_at api_date_format(petition.created_at)
+  json.updated_at api_date_format(petition.updated_at)
+  json.open_at api_date_format(petition.open_at)
+  json.closed_at api_date_format(petition.closed_at)
+  json.government_response_at api_date_format(petition.government_response_at)
+  json.scheduled_debate_date api_date_format(petition.scheduled_debate_date)
+  json.debate_threshold_reached_at api_date_format(petition.debate_threshold_reached_at)
+  json.rejected_at api_date_format(petition.rejected_at)
+  json.debate_outcome_at api_date_format(petition.debate_outcome_at)
+  json.moderation_threshold_reached_at api_date_format(petition.moderation_threshold_reached_at)
```

Comment:
Actually the api_date_format helper is used in the CSV generation so to keep it consistent I'll leave it